app/services/proofing/lexis_nexis/instant_verify/verification_request.rb

```diff
@@ -40,7 +40,7 @@ def formatted_address
             StreetAddress2: applicant[:address2] || '',
             City: applicant[:city],
             State: applicant[:state],
-            Zip5: applicant[:zipcode].match(/^\d{5}/).to_s,
```

Comment:
question (non-blocking): I feel like this is something we should be monitoring? I don't have a great handle on when we expect this value to be `nil`, if ever.

Reply:
Reading through previous comments, it seems my first question/concern has already been raised, which leaves the me with the second:
Do we ever expect zip to be `nil`? If we don't, then should we implement form validation for this field?

Reply:
@lmgeorge 💯%.

I was just talking to Matt H. about this and ended up putting this on the agenda for our next Tech Sync. It turns out this is a symptom of a bigger issue; a bit of an edge case, but one that people hit.